HTML/script.js

```diff
@@ -0,0 +1,3 @@
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2197085720>#099dce</a><span> - 06/28/2024, 02:39 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The event listener for 'DOMContentLoaded' is added directly in the global scope. This can lead to potential issues if multiple scripts are loaded and they all try to add event listeners in the same way, causing conflicts or unexpected behavior.
</div>
<div id="fix">
<b>Fix:</b> Encapsulate the event listener within an IIFE (Immediately Invoked Function Expression) to avoid polluting the global scope and potential conflicts with other scripts.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,3 +1,5 @@
 +(function() {
 +    document.addEventListener('DOMContentLoaded', function() {
 +        console.log('Document is ready!');
 +    });
 +})();
```
</div>
</details>
</div>
<br>


</div>